regolith/helpers/a_projectumhelper.py

```diff
@@ -122,7 +132,7 @@ def db_updater(self):
             'name': rc.name,
             'pi_id': rc.pi_id,
             'lead': rc.lead,
-            'notes': rc.notes,
+            'notes': rc.notes
```

Comment:
are any of these lines needed? I would delete them.

Reply:
These lines were already present in the file prior to my work on it. Is it okay to delete them?

Reply:
yes indeed.   You are doing a service to the group by making the code better!